driver-core/src/main/java/com/datastax/driver/core/TableOptionsMetadata.java

```diff
@@ -206,6 +218,19 @@ public String getComment() {
    * @return the read repair chance set for table (in [0.0, 1.0]).
    */
   public double getReadRepairChance() {
+    return readRepairChance;
+  }
+
+  /**
+   * Returns the read_repair option for this table. <b>NOTE:</b> this is a Cassandra® 4.0 and newer
+   * option (described here<a
```

Comment:
Nit: spurious word "here" before &lt;a> tag.

Reply:
Actually, I intended "here" to not be part of the link, but either way I've got it twice which was not the intent.